junit/MatrixTest.java

```diff
@@ -69,8 +69,21 @@ public void tooShort(){
         assertEquals(array2, m.getArray());
     }
     //Matrix(double[] vals, int m){}
-
-    //Matrix(int m, int n){}
+    
+    /**
+     * testMatrixConstructor3 -Tests the third matrix constructor as specified in issue #7.
+    **/
+    @Test
+    public void testMatrixConstructor3() {
```

Comment:
I was wondering this myself. @bclinthall what do you think about negative values or zero? Throw exception? Print error message to console?